### PR TITLE
Update for Chef Infra Client 14.x usage

### PR DESCRIPTION
**Pull request by sean-horn**
Signed-off-by <sean_horn@chef.io>

### Description

Make note of the best configuration for Chef Infra Client 14. x clients

### Check List

- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,23 @@ RuntimeError
 Audit Mode is enabled. The audit cookbook and Audit Mode cannot be used at the same time. Please disable Audit Mode in your client configuration.
 ```
 
+### Pre-modern Chef Infra Client and Inspec between 13.x - 14.x
```
Comment (tas50) [inline README.md:43]:
```suggestion
### Legacy Chef Infra Client Releases (pre-15.x) and Inspec
```

File: README.md [hunk 2]
```diff
@@ -40,6 +40,23 @@ RuntimeError
...
+
+Newer Chef Infra Clients, in the 15.x and 16.x ranges ship with an
+inspec-core gem that the audit cookbook in the proper version range
+listed in the matrix above knows to use.  It will not attempt to
```
Comment (tas50) [inline README.md:47]:
```suggestion
listed in the matrix above knows to use. It will not attempt to
```

File: README.md [hunk 3]
```diff
@@ -40,6 +40,23 @@ RuntimeError
...
+download a gem in this case.
+
+We find that 14.12 has the inspec-core 3.9.0 gem embedded which should
+generate reports compatible with A2, but anything before that uses
+InSpec gem v2.x which is NOT compatible with A2. To explain further,
+Inspec 2.x will send reports to A2, but they will false positive, as
+this report format is incompatible with A2
+
+Chef Client 14.x + plain, non inspec-core Inspec 3.9.3 installed by
```
Comment (tas50) [inline README.md:56]:
```suggestion
Chef Infra Client 14.x + plain, non inspec-core InSpec 3.9.3 installed by
```

**Review by kagarmoe — COMMENTED** [review]
We should get this merged. The fastest course of action is to take @sean-horn`s PR as-is and then have @tas50  add his changes in a separate PR.

**Pull request merged**